platform/core/src/utils/Combine.ts

```diff
@@ -0,0 +1,36 @@
+const handler = {
+  get: (target, prop, receive) => {
+    for (const obj of target) {
+      if (prop in obj) {
+        return obj[prop];
+      }
+    }
+  },
+
+  set: (target, prop, value) => {
+    target[0][prop] = value;
+    return true;
+  },
+
+  ownKeys: target => {
+    const ret = {};
+    for (const obj of target) {
+      if (!obj) {
+        continue;
+      }
+      for (const key of obj) {
+        ret[key] = key;
+      }
+    }
+    return Object.keys(ret);
+  },
+};
+
+/**
+ * Combine multiple objects into a single response object, just by "merging" the whole kit an caboodle.
+ * @param args to proxy as though they were a hierarchy
+ * @returns
+ */
+export default function Combine(...args) {
```

Comment:
If this is merge, it should be called Merge. Also, how is is it different than loadash merge https://lodash.com/docs/4.17.15#merge which is already implemented and tested heavily

Reply:
It is neither merge nor assign - it is actually a proxy dynamic combine, which is kind of like Object.assign or _.assign, but has different performance characteristics, appropriate for the use it is getting.  Probably I should have used merge to avoid premature optimization.